SU-SWS/stanford_publication
# D8CORE-3479: adjusting spacing for the Publication node page.

# READY FOR REVIEW
I merged with the 8.x-1.x and ended up with a lot of config changes which surprised me. I believe that is from the merge. But confirmation would be good.

# Summary
- Adjusting spacing on the publication node.
- Adding a line in Large and up viewports next to the right col citation information

# Needed By (Date)
- 2/11/21

# Urgency
- low

# Steps to Test

1. pull in this change.
2. refresh everything. 
3. notice the line between columns and adjusted spacing.

# Affected Projects or Products
- Publication module
- 
# Associated Issues and/or People
- D8CORE-3479
- https://github.com/SU-SWS/stanford_profile/pull/356

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
